src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -641,6 +691,51 @@ internal static string NormalizeFilePath(string path)
             return Path.GetFullPath(path);
         }
 
+        /// <summary>
+        /// Determine the execution policy based on the supplied string.
+        /// If the string doesn't match to any known execution policy, set it to incorrect.
+        /// </summary>
+        /// <param name="_executionPolicy">The value provided on the command line.</param>
+        /// <returns>The execution policy.</returns>
+        private static ParameterBitmap GetExecutionPolicy(string? _executionPolicy)
+        {
+            if (_executionPolicy is null)
+            {
+                return ParameterBitmap.EPUndefined;
+            }
+
+            ParameterBitmap ExecutionPolicySetting = ParameterBitmap.EPUndefined;
+            switch (_executionPolicy)
+            {
+                case var b when string.Equals(_executionPolicy, "default", StringComparison.OrdinalIgnoreCase):
```

Comment:
A minor comment: given that the var `b` is never used, it would be nicer to keep using `if`'s instead of a switch (the switch is actually also doing `if`'s in this case)
```
if (string.Equals(...))
{
    return xxx;
}

if (string.Equals(...))
{
    return xxx;
}

...
```